source/connectors/kafka-connect-migration.txt

```diff
@@ -0,0 +1,59 @@
+.. _kafka-connect-migration:
+
+==========================
+Migrate from Kafka Connect
+==========================
+
+.. default-domain:: mongodb
+
+This section lists the changes needed to migrate from the deprecated,
+community-developed version of the Kafka Connector (hosted in the
+`hpgrahsl/kafka-connect-mongodb GitHub repository
+<https://github.com/hpgrahsl/kafka-connect-mongodb>`_) to the `official
+connector <https://github.com/mongodb/mongo-kafka>`_.
+
+Once your project build file is updated to reference the official Kafka
+Connect library, use the following information to migrate your
+project.
+
+
+Package and Class Name Changes
+------------------------------
+
+- The package name for the Kafka Connect library used was changed from
+  ``at.grahsl.kafka.connect.mongodb`` to ``com.mongodb.kafka.connect``
+
+- The ``MongoDbSinkConnector`` class has been renamed ``MongoSinkConnector``
+
+- The ``MongoDbSinkConnectorConfig`` class has been split into
+  ``sink.MongoSinkConfig`` and ``sink.MongoSinkTopicConfig`` classes. Refer to
+  the source for a complete list of properties in `MongoSinkConfig
+  <https://github.com/mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/MongoSinkConfig.java>`_
+  and `MongoSinkTopicConfig
+  <https://github.com/mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/MongoSinkTopicConfig.java>`_.
```

Comment:
I wonder if this section would be clearer as a table with **New** and **Old** columns or somesuch

Reply:
I thought about that, but I'm not sure how I would represent items such as "remove the `mongodb.` prefix from all configurations" in a table like that other than attempting to identify every possible configuration that starts with `mongodb.` and creating a row for it.

Reply:
True that would be a little tricky. I primarily meant this section though (Package and Class Name Changes), not the Configuration Changes section.